### DISCUSSION TITLE
Is okey the documentation?

### DISCUSSION THREAD

**Discussion by UrielJavier** [originjs/vite-plugin-federation#155] [Q&A] [answered]
Hello!

Im making a POC with this plugin, and i saw that the documentation has an example of a remote microfront: 
'http://localhost:5011/remoteEntry.js',
but should be 'http://localhost:5011/dist/assets/remoteEntry.js'?

Tnks

**Comment by Jiannan-dev** [comment, marked as answer]
This depends on the exact location of the remoteEntry file generation and the configuration used, not on a fixed path

**Reply by UrielJavier** [reply]
Perfect! thnks
